### ISSUE TITLE
Create toolbar section for enrichment

### ISSUE THREAD

**Issue by CaitlinOCallaghan** [PathwayCommons/app-ui#800]
## Desired UI for toolbar 

![image](https://user-images.githubusercontent.com/38890251/40800312-06071cd6-64de-11e8-8222-65f950c67753.png)

## Task

Create toolbar section
- Title
- PC logo

**Comment by maxkfranz** [comment]
It might be a bit confusing with two visible search bars.  Should this new search bar just be an addition to the search page rather than the results page?

**Comment by CaitlinOCallaghan** [comment]
Max has a good point regarding the possible confusion. My only concern is that if the gene input is on a different page, then modifying the list could be inconvenient. We could collapse and expand the second search bar (like the live PC) since it is likely that users will only explore the tools section after already inputing their genes. But after looking at the design, I feel like it could be even more confusing. 

![image](https://user-images.githubusercontent.com/38890251/40842930-62bd62c4-657d-11e8-9643-2f3fdfd608e6.png)

Also, should the second search bar be for pathway names or for genes involved? I personally think a gene search to identify which pathways contain a specific gene would be more useful to users.

**Comment by jvwong** [comment]
The first bar ('Enter one gene') is really a data input for the pathway enrichment analysis. The second is more of a 'find'.  So it likely makes sense to remove the magnifying glass icon from the former.

**Comment by CaitlinOCallaghan** [comment]
Rather than a  'search' button with a magnifying glass, would we want to replace it with a 'submit' button? 

![image](https://user-images.githubusercontent.com/38890251/40854174-92b06596-659e-11e8-87a0-b2abe0309222.png)

**Comment by CaitlinOCallaghan** [comment]
This is what I have so far: 

![image](https://user-images.githubusercontent.com/38890251/40858265-c47d9bd6-65ab-11e8-8ead-39a662008aca.png)

The line "|" is a result from the parent base-network-view.js file rending.

Or we can format the title instead as:

![image](https://user-images.githubusercontent.com/38890251/40860003-b74d8006-65b1-11e8-868b-d34f432f0eb2.png)

**Comment by maxkfranz** [comment]
> My only concern is that if the gene input is on a different page, then modifying the list could be inconvenient. We could collapse and expand the second search bar (like the live PC) since it is likely that users will only explore the tools section after already inputing their genes.

How about hiding the entire toolbar until after the genes list has been entered?  I assume there's no graph until after the gene list is submitted, so the toolbar wouldn't be useful at first anyway.

Further, you can make the visibility of toolbar mutually exclusive with the visibility of the gene list entry.  You could show an edit/reset icon in place of the gene list UI so a user could reset/empty the graph to go back to the entry state.

Basically, it's the same idea as using separate pages but just working within the same page using state.

**Comment by jvwong** [comment]
Sure, this is a valid concern. Mutually exclusive sounds fine. 
My two cents: Be reasonable about how much time is required (let's not drop two weeks on this at the cost of moving forward).
